extensions/wikibase/module/styles/schema-alignment.css

```diff
@@ -183,6 +198,7 @@
   margin: 5px;
   white-space: normal;
   word-wrap: break-word;
+  padding: 10px;
```

Comment:
This sort of padding/margin increase seems unrelated to the original issue, no? What was your motivation to make this change?

Reply:
In the original code, the padding in the Terms and Statements input boxes were not equal,
![image](https://user-images.githubusercontent.com/108210025/224537449-a2d9e52d-eb88-4a76-87c1-8906b815ef67.png)

 Which is why I took the liberty of adding padding to the Statement input box.
But yes, I agree it is unrelated to the original issue. I'll remove that line

Reply:
Ok, it's a sensible move indeed but let's keep it out this time for the sake of having a clearer PR